spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,10 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      issuer = 'http://localhost:3000'
+      user = sign_in_and_2fa_user
+      loa3_sp_session(issuer: issuer)
```

Comment:
Per my above comment, its expected to be a likely scenario. I agree it would be good to have a fallback but am not 100% sure what that would look like right now.